lib/kennel/models/slo.rb

```diff
@@ -4,7 +4,10 @@ module Models
     class Slo < Record
       include TagsValidation
 
-      READONLY_ATTRIBUTES = superclass::READONLY_ATTRIBUTES + [:type_id, :monitor_tags, :target_threshold, :timeframe, :warning_threshold]
+      READONLY_ATTRIBUTES = [
+        *superclass::READONLY_ATTRIBUTES,
+        :type_id, :monitor_tags, :target_threshold, :timeframe, :warning_threshold
+      ].freeze
```

Comment:
this missing freeze was hiding the issue with the importer removing sli_specification during normalize (we added sli_specification to READONLY_ATTRIBUTES in a local hack)